kyuubi-events/src/main/scala/org/apache/kyuubi/events/EventLoggerType.scala

```diff
@@ -22,5 +22,5 @@ object EventLoggerType extends Enumeration {
   type EventLoggerType = Value
 
   // TODO: Only SPARK is done now
-  val SPARK, JSON, JDBC, CUSTOM = Value
+  val SPARK, JSON, JDBC, KAFKA, CUSTOM = Value
```

Comment:
Should we add enumeration values instead of modifying the order?

Reply:
`KAFKA` value is added before `CUSTOM` here, in order to 
1. follow the same order as in docs
2. `CUSTOM` is an extra option, which could be listed behind the options with embedded support

I'm open to your suggestion. WDYT @pan3793 ?

Reply:
Let's add the new one at the tail, to suppress the potential serializable issue. And the TODO comment is outdated

Reply:
SGTM. Let me move it to the tail, as you mentioned serializable problem.